src/App.tsx

```diff
@@ -1,61 +1,104 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { Todo } from './Types/TodoType';
+
+import { TodoList } from './components/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const addTodo = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    const isTitleValid = title.trim() !== '';
+    const isUserValid = userId !== 0;
+
+    setTitleError(!isTitleValid);
+    setUserError(!isUserValid);
+
+    if (!isTitleValid || !isUserValid) {
+      return;
+    }
+
+    const maxId = Math.max(...todos.map(todo => todo.id));
```

Comment:
You can create func with the name generateId or something like that(as helper), then use it in places were you need it.